packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -810,6 +811,9 @@ export interface CallWithChatControlOptions extends CommonCallControlOptions {
 // @public
 export type CallWithChatEvent = 'callError' | 'chatError' | 'callEnded' | 'isMutedChanged' | 'callIdChanged' | 'isLocalScreenSharingActiveChanged' | 'displayNameChanged' | 'isSpeakingChanged' | 'callParticipantsJoined' | 'callParticipantsLeft' | 'selectedMicrophoneChanged' | 'selectedSpeakerChanged' | /* @conditional-compile-remove(close-captions) */ 'isCaptionsActiveChanged' | /* @conditional-compile-remove(close-captions) */ 'captionsReceived' | 'messageReceived' | 'messageSent' | 'messageRead' | 'chatParticipantsAdded' | 'chatParticipantsRemoved';
 
+// @beta
+export type CallWithChatOptions = ChatAdapterOptions;
```

Comment:
@PorterNan So this is what it would look like if we introduced a new options. But since we don't have a `CallAdapterOptions` yet, it isn't a true union yet. So I don't know, when we do create a `CallAdapterOptions` will this be considered a breaking change?

Reply:
Just check the code, looks like we have AzureCommunicationCallAdapterOptions, I guess that is a good hint that we should align chatAdapter option name (a name change, we can do it in this PR)

And in the meantime, back to your question - if CallAdapterOption or AzureCommunicationCallAdapterOptions is not needed for now, let's directly use ChatAdapterOptions or AzureCommunicationChatAdapterOptions as the option type, then we can add

export type CallWithChatOptions = AzureCommunicationChatAdapterOptions && AzureCommunicationCallAdapterOptions; later when it is needed, changes like this is not considered as a breaking change